pkgs/by-name/cr/cronutils/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cronutils";
+  version = "1.10";
+
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "cronutils";
+    rev = "version/${version}";
+    hash = "sha256-XJksfX4jqE32l4HipvO26iv9W4c0Iss6DenlEatdL1k=";
+  };
+
+  makeFlags = [ "DESTDIR=$(out)" "prefix=" ];
```

Comment:
Please see: https://github.com/NixOS/nixpkgs/issues/65718

Reply:
I've removed `DESTDIR` and set `prefix=$(out)`, I hope I got that right from the thread.